metricflow-semantics/metricflow_semantics/specs/patterns/entity_link_pattern.py

```diff
@@ -136,13 +120,10 @@ def match(self, candidate_specs: Sequence[InstanceSpec]) -> Sequence[LinkableIns
         # Entity links could be a partial match, so it's handled separately.
         if ParameterSetField.ENTITY_LINKS in self.parameter_set.fields_to_compare:
             filtered_candidate_specs = self._match_entity_links(filtered_candidate_specs)
-        # Time granularities are special, so they are also handled separately.
```

Comment:
So we just won't check this field anymore ?

Reply:
We check it via the standard comparison, see the change below to the set.difference() call.